<ui version="4.0" >
 <class>SqlItemView</class>
 <widget class="QWidget" name="SqlItemView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item rowspan="6" row="0" column="0" >
    <widget class="QScrollArea" native="1" name="scrollWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" />
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QPushButton" name="firstButton" >
     <property name="toolTip" >
      <string>Move to the first record</string>
     </property>
     <property name="text" >
      <string>&lt;&lt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="previousButton" >
     <property name="toolTip" >
      <string>Move to the previous record</string>
     </property>
     <property name="text" >
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="nextButton" >
     <property name="toolTip" >
      <string>Move to the next record</string>
     </property>
     <property name="text" >
      <string>></string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QPushButton" name="lastButton" >
     <property name="toolTip" >
      <string>Move to the last record</string>
     </property>
     <property name="text" >
      <string>>></string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <widget class="QLabel" name="positionLabel" >
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QScrollArea</class>
   <extends>QWidget</extends>
   <header location="global" >QScrollArea</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
